pontoon/contributors/utils.py

```diff
@@ -427,24 +427,89 @@ def get_project_locale_contribution_counts(contributions_qs):
     return counts
 
 
-def get_contribution_timeline_data(user, contribution_type=None, day=None):
+def get_contribution_timeline_data(
+    user, year_shown=None, contribution_type=None, day=None
+):
     """
     Get data required to render the Contribution timeline on the Profile page
     """
     end = timezone.now()
-    start = end - relativedelta(months=1)
-    timeline_title = "Contribution activity in the last month"
+
+    if not year_shown:
+        start = end - relativedelta(day=1)
+        timeline_title = "Contribution activity in this month"
+    else:
+        timeline_title = "Contribution activity in the past year"
```

Comment:
If there's no clear reason why to test `if not year_shown`, the test should be `if year_shown`.